packages/utilities/gel-foundations-styled-components/README.md

```diff
@@ -0,0 +1,31 @@
+# gel-foundations-styled-components &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-foundations-styled-components.svg)](https://www.npmjs.com/package/@bbc/gel-foundations-styled-components) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection of styled components which can be imported into your application.
```

Comment:
Again, not hugely descriptive. My understanding: this package is intended to contain the GEL grid & typography. On the whole "shareability" front, I think we need to be really, really good about how we name, document & present our packages to people who've never seen them before if we ever want to convince them to use them. 🤔

Reply:
Talked on slack, made a change but happy for @bcmn to suggest a change